From 7253dc1b5c2ab520dd15887fe4847b5615ad3e26 Mon Sep 17 00:00:00 2001
From: Yuta Saito <kateinoigakukun@gmail.com>
Date: Fri, 26 May 2023 16:19:37 +0000
Subject: [PATCH] [test] Add missing default value for target_run_with_leaks

This property has been reverted in main branch but still exists in this
release branch.
Reverting commit 401e9304f41e0179f631f2e8fbed30b8b20769fc
---
 test/lit.cfg | 1 +
 1 file changed, 1 insertion(+)

diff --git a/test/lit.cfg b/test/lit.cfg
index e229dc85a09..8a955f0da88 100644
--- a/test/lit.cfg
+++ b/test/lit.cfg
@@ -1082,6 +1082,7 @@ target_specific_module_triple = config.variant_triple
 target_future = target_specific_module_triple
 
 config.target_run = ''
+config.target_run_with_leaks = ''
 config.target_rtti_opt = '-fno-rtti'
 config.target_pic_opt = ''
 config.target_cxx_lib = '-lc++'
-- 
2.40.0

